app/src/main/java/edu/uc/forbesne/shoppingsidekick/service/ProductService.kt

```diff
@@ -84,7 +84,7 @@ class ProductService {
         return _products
     }
 
-    fun fetchAllProductsFromThreeStore() : MutableLiveData<ArrayList<Product>> {
+    internal fun fetchAllProductsFromThreeStore() : MutableLiveData<ArrayList<Product>> {
```

Comment:
I added an internal visibility modifier in Product Service. This was recommended in best practices site for more coherency. (https://phauer.com/2017/idiomatic-kotlin-best-practices/)